### PR TITLE
update Connect page title and remove some page_title in Chinese to be consistent with Eng page

### LINKED ISSUE #1008: Jakarta EE Connect page minor update
https://jakarta.ee/connect/ has Jakarta EE Monthly Update Calls section that should be renamed to Jakarta EE Community Calendar. Can you please make this update?

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #1008 

Hi @laliwa Eric, do you think this translation needs to be updated for this minor title change? Thanks!

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

File: content/compatibility/_index.zh.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ hide_sidebar: true
 draft: false
 links: [[href: "get_listed", text: "加入列表"]]
 layout: "single"
+hide_page_title: true
+container: "container margin-top-20"
```
Comment (flora8984461) [inline content/compatibility/_index.zh.md:11]: This is just hope to make consistent with Eng page

File: content/connect/_index.zh.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ date: 2018-04-05T16:10:38-04:00
 description: "Jakarta EE 工作组最新进展、社群新闻和公告."
 hide_sidebar: "true"
 layout: "single"
+hide_page_title: true
```
Comment (flora8984461) [inline content/connect/_index.zh.md:8]: This is just hope to make consistent with Eng page

File: content/resources/_index.zh.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ headline: "资源"
 date: 2019-02-27T16:10:38-04:00
 description: "Jakarta EE 资源"
 layout: "single"
+hide_page_title: true
```
Comment (flora8984461) [inline content/resources/_index.zh.md:7]: This is just hope to make consistent with Eng page

**Comment by laliwa** [comment]
> Fix #1008
> 
> Hi @laliwa Eric, do you think this translation needs to be updated for this minor title change? Thanks!
> 
> Signed-off-by: Yi Liu [yi.liu@eclipse-foundation.org](mailto:yi.liu@eclipse-foundation.org)

Sure, I made a PR here https://github.com/jakartaee/jakarta.ee/pull/1011

**Pull request merged**